src/test/java/com/jcabi/http/wire/LastModifiedCachingWireTest.java

```diff
@@ -65,6 +68,11 @@
     private static final String BODY_UPDATED = "Test body updated";
 
     /**
+     * Test body updated 2.
+     * */
+    private static final String BODY_UPDATED_2 = "Test body updated 2";
```

Comment:
@mkordas I will do this, it will now not match the style already in these tests. I will raise todos for those.